sdk/cosmosdb/cosmos/src/client/ChangeFeed/buildChangeFeedIterator.ts

```diff
@@ -86,6 +92,28 @@ export function changeFeedIteratorBuilder(
     );
     const cfResource = changeFeedStartFrom.getCfResource();
     if (isPartitionKey(cfResource)) {
+      const partitionKey = cfResource as PartitionKey;
+      const partitionKeyDefinition = await container.getPartitionKeyDefinition();
+
+      if (
+        partitionKeyDefinition !== undefined &&
+        isPrefixPartitionKey(partitionKey, partitionKeyDefinition.resource)
```

Comment:
What happens if empty value of partition key array is passed (Eg. [])
Are we now setting epkRange for the entire container?

Reply:
Yes correct. I also tested this. So we can say ,
changeFeedStartFrom: ChangeFeedStartFrom.Beginning(),
changeFeedStartFrom: ChangeFeedStartFrom.Beginning([]),
both if these above two lines behave same, covering the entire container.